classes/Avatar.php

```diff
@@ -165,6 +166,8 @@ public static function getUserHtml($intId, $intWidth=null, $intHeight=null)
         if ($strFile == '') {
             return '';
         }
+        $objUser = \MemberModel::findByPk($intId);
```

Comment:
In that function you cannot use the MemberModel, becuase this function is only for Users, which are Backend Users in Contao. So i think you have to find a different Alt handling for Users.